pkgs/applications/networking/instant-messengers/skypeforlinux/default.nix

```diff
@@ -59,7 +59,10 @@ let
   src =
     if stdenv.hostPlatform.system == "x86_64-linux" then
       fetchurl {
-        url = "https://repo.skype.com/deb/pool/main/s/skypeforlinux/skypeforlinux_${version}_amd64.deb";
+        urls = [
+          "https://repo.skype.com/deb/pool/main/s/skypeforlinux/skypeforlinux_${version}_amd64.deb"
+          "https://web.archive.org/web/https://repo.skype.com/deb/pool/main/s/skypeforlinux/skypeforlinux_${version}_amd64.deb"
+        ];
```

Comment:
I have no trouble with this PR. I wonder how Archive.org feels about following the license of Skype, which I assume says you can't redistribute it :)

Reply:
There is some precedent for using this approach in https://github.com/NixOS/nixpkgs/issues/70155 and a few other places for unreliable upstreams. But yes, indeed the big question remains, what does this actually mean in legal terms for archive.org. Maybe @fpletz knows sth.

In general, longer term, I'd almost prefer if we would move out a lot of those nastier non-free things out in some way that still allows the users to use nix to install it, but versions and urls and what not would not sth that we manage in the core repo (like NUR/flakes). Because otherwise either we keep chasing versions or deal with what we see here.

Reply:
From ToS (https://www.skype.com/en/legal/ios/tos/ relevant section 4):

```
For the avoidance of doubt, (a) IT administrators working on behalf of a company may download and install the Skype Software onto personal computers or other devices used by personnel of such company; and (b) you are permitted to use the Software at a university or other educational institution, or at work. Skype reserves all rights not expressly granted to you under these Terms.
```

Applying not-lawyer-reasoning, apt-mirrors would be at least fine until those terms.